js/bibox.js

```diff
@@ -2107,6 +2109,81 @@ module.exports = class bibox extends Exchange {
         };
     }
 
+    async fetchDepositWithdrawFee (code, params = {}) {
```

Comment:
should be plural here right? fetchDepositWithdrawFee**s** ?

Reply:
> should be plural here right? fetchDepositWithdrawFee**s** ?

No, a currency is required, should it should be singular

Reply:
@samgermain yeah, but if we're going to return deposit+withdraw fees (more than one fee)  it could be `fees` as well

Reply:
> @samgermain yeah, but if we're going to return deposit+withdraw fees (more than one fee) it could be `fees` as well

You could think of it as fetch the deposit fee and the withdraw fee, if we do it like how you're saying, we'll need to change the name of the method `fetchDepositWithdrawFees`

Reply:
@samgermain alright if the team agrees that's ok with me as well